### PR TITLE
Adding (CC BY-NC-ND) license

### PR DESCRIPTION
**Pull request by mateus-fernandes20**
Adding license to Algoritmos e Estruturas de Dados 1 - Marcos Castilho, Fabiano Silva, Daniel Weingaertner (PDF)

## What does this PR do?
Add info based on https://github.com/EbookFoundation/free-programming-books/issues/11355

### Description
* [Algoritmos e Estruturas de Dados 1](https://www.inf.ufpr.br/marcos/livro_alg1/livro_alg1.pdf) - Marcos Castilho, Fabiano Silva, Daniel Weingaertner (PDF)
License (CC BY-NC-ND) found on PDF page 3

### For book lists, is it a book? For course lists, is it a course? etc.
* It's a book
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -387,7 +387,7 @@
 
 ### Pascal
 
-* [Algoritmos e Estruturas de Dados I](https://www.inf.ufpr.br/marcos/livro_alg1/livro_alg1.pdf) - Marcos Alexandre Castilho, Fabiano Silva, Daniel Weingaertner (PDF)
+* [Algoritmos e Estruturas de Dados 1](https://www.inf.ufpr.br/marcos/livro_alg1/livro_alg1.pdf) - Marcos Castilho, Fabiano Silva, Daniel Weingaertner (PDF) (CC BY-NC-ND)
```
Comment (eshellman) [inline books/free-programming-books-pt_BR.md:390]: Looks like this is a duplicate listing! we only allow one entry per resource. Thanks for noticing!! (please remove it)

Comment (mateus-fernandes20) [reply books/free-programming-books-pt_BR.md:390]: Sure! I'll change right away.

**Review by mateus-fernandes20 — COMMENTED** [review]
Removed duplicated listing and added (CC BY-NC-ND) license.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**